Hometask8/src/main/java/homework21/CheckoutStepOnePage.java

```diff
@@ -1,14 +1,26 @@
 package homework21;
 
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Condition.text;
 import static com.codeborne.selenide.Selenide.$;
 
 public class CheckoutStepOnePage {
-    static String[] selectors = {"#first-name", "#last-name", "#postal-code"};
-    public static void fillCheckoutInformation(String[] data) {
-        for(int i = 0; i < data.length; i++) {
-            $(selectors[i]).setValue(data[i]);
-        }
+    static SelenideElement title = $(".title");
+    static final String checkoutTitle = "Checkout: Your Information";
+    static SelenideElement continueBtn = $("#continue");
+
+    public static void fillCheckoutInformation() {
+        $("#first-name").setValue("Kate");
+        $("#last-name").setValue("Smith");
+        $("#postal-code").setValue("98765");
```

Comment:
Passed data should not be hardcoded, make it like
fillCheckoutInformation(String firstname, String lastname, String postal)